src/rez/solver.py

```diff
@@ -74,9 +97,6 @@ class SolverStatus(Enum):
     cyclic = ("The solve contains a cycle.", )
     unsolved = ("The solve has started, but is not yet solved.", )
 
-    def __init__(self, description):
-        self.description = description
-
```

Comment:
mypyc did not like adding a new `__init__` for `Enum`, so it was simple enough to replace this attribute with a call to the enum `value`.